docs/blog/2022-11-21-epss.mdx

```diff
@@ -91,20 +103,27 @@ That's too bad for us, because one of the problems we are trying to solve with [
 is tuning our vulnerability prioritization to consider the environment the code is running in. We are working hard to build our own source
 of signal for that.
 
-### The result is opaque
-CVSS scores are calculated based on a limited set of inputs and the mathematical formula can be evaluated by a human.
+### The results are opaque
+
+CVSS scores are calculated based on a limited set of inputs and the mathematical formula can be manually computed with a calculator in just a few minutes.
+
+In contrast the opaque machine learning based approach of EPSS is very resistant to manual human calculation.
+The model is not Open Source and it is more difficult
+to introspect the results. The only output ever surfaced to consumers is the EPSS score itself, a value between 0 and 1.
+
+## Is EPSS relevant?
+
+As far as we can tell, EPSS is a good idea and is a much needed step that helps companies effectively prioritize their security efforts.
 
-In contrast, the machine learning based approach of EPSS is more opaque. The model is not open source, and it is more difficult
-to introspect the results. The only output of the model surfaced to consumers is the EPSS score itself. Previous versions
-of the model were detailed in academic papers
+But, while it's an exciting system that will eventually help solve the problem of too many CVEs being [false positives](https://www.lunasec.io/docs/blog/the-issue-with-vuln-scanners/), it's still early in it's maturity and very few security tools even include it today. There is also more validation required before security experts are willing to put their weight behind recommending that companies or standard use EPSS[^4].
 
-## Is EPSS relevant
+## How to try out EPSS
 
-The shortcomings of CVSS and existing scanning tools are pretty well known, and in fact, we are trying to build a business
-disrupting that. As far as we can tell, EPSS is a good idea and a needed step on the way to less bogus work for security engineers.
+While EPSS is only officially available as an API, we've been working to add EPSS support into the security tool we've been building on GitHub[^5][^6] that aggregates vulnerability data in one place.
```

Comment:
This is a downgrade from what I wrote. You are presenting our tool as something that "aggregates vulnerability data" while that is really a sunset of what it does.  This sentence should probably roll back. This is our core hook for our product. It is not a "vulnerability data aggregator", that's boring and shallow compared to our goal